### ISSUE TITLE
namedPlaceHolders doesn't seem to work with manual prepared statement

### ISSUE THREAD

**Issue by midnightcodr** [sidorares/node-mysql2#756]
Hi,
I am getting

>   code: 'ER_PARSE_ERROR',
>   errno: 1064,

while trying to use [manual prepared statement](https://github.com/sidorares/node-mysql2/blob/master/documentation/Prepared-Statements.md) with a connection that enables `namedPlaceHolders`. Thoughts?

**Comment by sidorares** [comment]
You are right: https://github.com/sidorares/node-mysql2/blob/5c5aa3aed7e772e599dbefdafb04405672901808/lib/connection.js#L640

It's actually a good place for first contribution. Would you like to give it a go? If not that's fine, I can have a look over weekend

**Comment by midnightcodr** [comment]
Would love to but it might take some time. For now I'll just use cached prepared statement. Thanks for the quick response.
